authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,43 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
 
-"""Configure JWT Here"""
+from django.conf import settings
 
+from rest_framework import authentication, exceptions
+
+from .models import User
+
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    authentication_header_prefix = 'Bearer'
+
+    def authenticate(self, request):
+        """Authenticate the user """
+        request.user = None
+        # get a list of the items in the header
+        auth_header = authentication.get_authorization_header(request).split()
+        auth_header_prefix = self.authentication_header_prefix.lower()
+
+        if len(auth_header) == 2:
+            prefix = auth_header[0].decode('utf-8')
+            token = auth_header[1].decode('utf-8')
+            if prefix.lower() != auth_header_prefix:
+                msg = 'Bearer is expected as the prefix '
+                raise exceptions.AuthenticationFailed(msg)
+            else:
+                return self._authenticate_credentiatials(request, token)
+        else:
+            msg = 'The header must contain only two elements'
+            raise exceptions.AuthenticationFailed(msg)
```

Comment:
@VivianDoreen Change this to fix the tests.
```suggestion
            return None
```

Reply:
Ok, let me do that

Reply:
I have made the necessary changes. kindly review